src/api/app/models/obs_factory/staging_project.rb

```diff
@@ -178,6 +169,7 @@ def missing_reviews
         attribs = [:by_group, :by_user, :by_project, :by_package]
 
         (open_requests + selected_requests).uniq.each do |req|
+          package = req.bs_request_actions.first.target_package
```

Comment:
you have this now in 3 places. Once in this model and twice in views - not worth a model function ?

Reply:
I thought about it, but what would be the name, `first_target_package`? :thinking: I guess we could do that.

Reply:
you can also name it 'nick_name' - and possibly make it smarter over time :)

As maintenance requests would use a different nick name.

Reply:
I think what I didn't like about it is that it feels a bit arbitrary to make the first request action special. But it seems to be a thing. So let's add a helper method.